test/unit/spec/v2/view-builder/utils/LinksUtil_spec.js

```diff
@@ -0,0 +1,42 @@
+import AppState from 'v2/models/AppState';
+import { FORMS } from 'v2/ion/RemediationConstants';
+import { getSwitchAuthenticatorLink } from 'v2/view-builder/utils/LinksUtil';
+
+describe('v2/utils/LinksUtil', function() {
+  const mockAppState = (remediationFormName, hasMoreThanOneAuthenticator) => {
+    const appState = new AppState();
+    spyOn(appState, 'getRemediationAuthenticationOptions').and.callFake(formName => {
```

Comment:
Similar to the other changes, let's make sure there are no `spyOn` methods being called that aren't methods on `jest`. This helps us eventually migrate to future major versions of Jest, [which plans](https://jestjs.io/blog/2020/05/05/jest-26#a-new-way-to-consume-jest---jestglobals) to eventually drop these as a part of their global objects.

```suggestion
    jest.spyOn(appState, 'getRemediationAuthenticationOptions').mockImplementation(formName => {
```